### ISSUE TITLE
A typo in main-cp.py?

### ISSUE THREAD

**Issue by MashShinozaki** [YatingMusic/compound-word-transformer#2]
Hi, this is not a crucial problem at all, but I found a typo in the code of main-cp.py.
The lines from 623 to 634 are
```
loss = epoch_loss
if 0.4 < loss <= 0.8:
    fn = int(loss * 10) * 10
    saver_agent.save_model(net, name='loss_' + str(fn))
elif 0.05 < loss <= 0.30:
    fn = int(loss * 100)
    saver_agent.save_model(net, name='loss_' + str(fn))
elif loss <= 0.05:
    print('Finished')
    return  
else:
    saver_agent.save_model(net, name='loss_high')
```
This means that, when the loss is between 0.3 and 0.4, the model is saved as 'loss_high', not as 'loss_31', 'loss_35' etc.
So, the first elif condition should be like this?
```elif 0.05 < loss <= 0.40:```

**Comment by wayne391** [comment]
Hi, I've fixed it. 
Thanks for your help.
